administrator-documentation/moderne-platform/how-to-guides/lst-cleanup.md

```diff
@@ -0,0 +1,36 @@
+# Cleaning up old LSTs
+
+As your team develops new code, LSTs will be created and added to your artifact repository. Over time, however, many of these LSTs will no longer be useful. This is especially common when working in feature branches that get merged in or deleted – all LSTs for those branches serve no point as any changes made on them can't be merged in at that point.
+
+While this isn't a big deal, it does cost your company money to continue to store these defunct LSTs in your artifact repository. Because of that, we encourage you to create some automation that removes old LSTs (such as ones that haven't been updated in more than week).
+
+An important caveat with this is that Moderne **does not** poll for artifacts being deleted. Once Moderne has downloaded an LST, it will continue to allow you to run recipes on it - even if those artifacts no longer exist in your artifact repository. Trying to commit changes against these LSTs will just result in errors.
+
+To address that issue, you have two main options: 
+
+1. A GraphQL cleanup query to sync all LSTs
+2. Writing custom, more targeted code to sync specific LSTs
```

Comment:
First step is actually to delete the obsolete LSTs from your artifact repository. Without first deleting them there reindex will have no effect. In fact, running it will _bring back_ repository entries you might have deleted from moderne with a separate mutation but have not deleted the corresponding LST artifact.